clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy

```diff
@@ -68,8 +74,12 @@ class Main extends SpringBootServletInitializer {
   }
 
   static void main(String... args) {
+    BOOTSTRAP_SYSTEM_PROPS.each { key, value -> System.setProperty(key, value) }
```

Comment:
if these are defaults, they should only setProperty if there isn't already a value for that property (otherwise I suspect its impossible to change for example bootstrap.location)

Reply:
I can make that change